packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -267,6 +299,16 @@ export default class DatabaseService {
       );
   }
 
+  getVRack(projectId) {
+    return this.$http
+      .get(
+        `/cloud/project/${projectId}/vrack`,
+        DatabaseService.getIcebergHeaders(),
+      )
+      .then(({ data }) => data)
+      .catch((error) => (error.status === 404 ? [] : Promise.reject(error)));
```

Comment:
angular-js uses $q and isn't bound to es6 Promises so $q.reject instead of Promise

Reply:
Fixed